Server/src/main/java/net/minecrell/serverlistplus/server/ServerListPlusServer.java

```diff
@@ -238,8 +224,9 @@ public int getProtocolVersion() {
         // Favicon
         FaviconSource favicon = response.getFavicon();
         if (favicon != null && favicon != FaviconSource.NONE) {
-            Optional<String> icon = faviconCache.getUnchecked(favicon);
-            icon.ifPresent(ping::setFavicon);
+            Optional<String> icon = faviconCache.get(favicon).toJavaUtil();
```

Comment:
Can you drop `.toJavaUtil()` now? I guess it might result in having to fully qualify the Guava Optional?